src/components/react-admin/DeleteWithConfirmButtonJS.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { fade } from '@material-ui/core/styles/colorManipulator';
+import ActionDelete from '@material-ui/icons/Delete';
+
+import {
+  Button,
+  Confirm,
+  useDeleteWithConfirmController,
+  useResourceContext,
+  useTranslate,
+} from 'react-admin';
+
+import classnames from 'classnames';
+import inflection from 'inflection';
```

Comment:
Where is `inflection` package? 
Did you forget to commit its installation in the package.json (and lock)?

Reply:
I did it